docs/source/DeveloperDocumentation/MobileSdks/ios-fairplay-sdk.md

```diff
@@ -100,13 +100,13 @@ To provide complete configuration for each `Stream` object in the `Streams.plist
  - A `name` for the content. This will be the readable display name and is not used by the SDK. This may differ from the `content_id`.
  - A `content_id` for the content, being the same content ID that the content was prpeared with. This value will always be the last path component of the stream `content_key_id_list` or `skd://` URI entry. Our demo application shows how to retrieve and parse the `content_key_id_list`, however, the `content_id` always needs to be provided for use with offline assets.
  - An `is_protected` boolean value which indicates that the `Stream` object is protected with DRM. This setting would not be needed in scenarios where all content is protected, as all assets can be set as protected by default in the source code. 
- - A `vudrm-token` which must be correctly generated for the type of instance you wish to create. Example VUDRM token type templates available are:
+ - A `vudrm-token` which must be correctly generated for the type of instance you wish to create. Example Studio DRM token type templates available are:
```

Comment:
we have to keep `vudrm-token` right?

Reply:
Yep, unfortunately that will have to stay as it is referencing values in the actual code base.

Reply:
Got it. thanks!